lib/api/streaming.js

```diff
@@ -169,25 +224,37 @@ Streaming.prototype.channel = function(channelId) {
  * Subscribe topic/channel
  *
  * @param {String} name - Topic name
+ * @param {String} clientType - Client type
  * @param {Callback.<Streaming~StreamingMessage>} listener - Streaming message listener
  * @returns {Subscription} - Faye subscription object
  */
-Streaming.prototype.subscribe = function(name, listener) {
-  var channelName = name.indexOf('/') === 0 ? name : '/topic/' + name;
-  var fayeClient = this._getFayeClient(channelName);
+Streaming.prototype.subscribe = function(name, clientType, listener) {
```

Comment:
This argument addition will raise incompatibility with old interface of `Stream#subscribe`.
We should not introduce such a hard as possible as we can.